packages/app/src/client/utils/request.ts

```diff
@@ -19,7 +19,7 @@ import * as rax from "retry-axios";
 
 const request = axios.create({
   // For config options see https://github.com/axios/axios#request-config
-  timeout: 5000
+  timeout: 10000
```

Comment:
For the record: we discussed the numbers for two specific HTTP requests. 

This change applies globally where this HTTP client is used -- I assume this means: also for other HTTP requests than the two discussed here.

Let's merge this now.